file-formats/ront/type/zif_aff_ront_v1.intf.abap

```diff
@@ -0,0 +1,60 @@
+INTERFACE zif_aff_ront_v1
+  PUBLIC.
+
+  CONSTANTS:
+    BEGIN OF co_category,
+      "! <p class="shorttext">Business Object</p>
+      "! Business object
+      business_object      TYPE c LENGTH 2 VALUE `BO`,
+      "! <p class="shorttext">Technical Object</p>
+      "! Technical object
+      technical_object     TYPE c LENGTH 2 VALUE `TO`,
+      "! <p class="shorttext">Analytical Object</p>
+      "! Analytical object
+      analytical_object    TYPE c LENGTH 2 VALUE `AO`,
+      "! <p class="shorttext">Configuration Object</p>
+      "! Configuration object
+      configuration_object TYPE c LENGTH 2 VALUE `CO`,
+    END OF co_category.
+
+  TYPES:
+    "! <p class="shorttext">Node</p>
+    BEGIN OF ty_rontnode,
+      "! <p class="shorttext">Name</p>
+      "! Raw (camel case) node name
+      "! $required
+      node_name_raw TYPE c LENGTH 40,
+
+      "! <p class="shorttext">Description</p>
+      "! Description
+      "! $required
+      description   TYPE zif_aff_types_v1=>ty_description_60,
+    END OF ty_rontnode.
```

Comment:
Minor: Does it make sense to rename the type to `ty_node_type`?

```suggestion
    BEGIN OF ty_node_type,
      "! <p class="shorttext">Name</p>
      "! Raw (camel case) node name
      "! $required
      node_name_raw TYPE c LENGTH 40,

      "! <p class="shorttext">Description</p>
      "! Description
      "! $required
      description   TYPE zif_aff_types_v1=>ty_description_60,
    END OF ty_node_type.
```

Reply:
Yes, I think that's a good idea. It's more consistent with the component names root_node_type and node_types and is easier to understand for someone looking at the interface for the first time. Thanks!